src/openproduct/producttypen/models/validators.py

```diff
@@ -34,6 +36,13 @@ def validate_thema_gepubliceerd_state(hoofd_thema, gepubliceerd, sub_themas=None
         )
 
 
+def validate_producttype_code(value):
+    if not re.match(r"^[A-Z0-9-]+$", value):
+        raise ValidationError(
+            _("Code mag alleen hoofdletters, cijfers en koppeltekens bevatten."),
+        )
```

Comment:
```suggestion
validate_producttype_code = CustomRegexValidator(
    regex="^[A-Z0-9-]+$",
    message=_(
        "Code mag alleen hoofdletters, cijfers en koppeltekens bevatten."
    ),
)
```
Could you change it to using  CustomRegexValidator from utils? This way the regex is show in the api spec.